site/docs/fargate-tutorials/prisma.md

```diff
@@ -235,15 +234,13 @@ curl ${QUICKSTART_LB_DNS}:8088/health
 Delete the resources created by this tutorial using the following SQL code:
 
 ```sql title="psql postgres://qpp3pzqb:LN6jnHfhRJTBD6ia@db.iasql.com/_3ba201e349a11daf -c"
+SELECT iasql_begin();
 DELETE FROM repository_image WHERE private_repository_id = (SELECT id FROM repository WHERE repository_name = 'quickstart-repository');
 DELETE FROM ecs_simplified WHERE app_name = 'quickstart';
+SELECT iasql_commit();
 ```
 
-Apply the changes described in the hosted db to your cloud account
-
-```sql title="psql postgres://qpp3pzqb:LN6jnHfhRJTBD6ia@db.iasql.com/_3ba201e349a11daf -c"
- SELECT * from iasql_apply();
-```
+The `iasql_begin()` and `iasql_commit()` functions are IaSQL RPCs that are used to start and then end a transaction. We use those two functions to push changes to the cloud.
```

Comment:
As a general comment: I'd prefer cases where we no longer *need* to wrap in a transaction in the documentation to just not do so. It needlessly complicates things and may put people under the impression that they're required when they aren't.

Reply:
That makes sense. There can be a comment pointing them to transactions in case they want to make the changes in a blocking fashion